### PR TITLE
Retry bulk delayed jobs

### PR DESCRIPTION
**Pull request by roggervalf**
In order to trigger delayed jobs in bulk action.

### CODE REVIEW

**Comment by bradvogel** [comment]
Are you sure it's safe to retry delayed jobs in all both Bull and Bee Queue? In looking at Bee Queue's code, it looks like it expects the job to be in the failed state.

**Comment by roggervalf** [comment]
> Are you sure it's safe to retry delayed jobs in all both Bull and Bee Queue? In looking at Bee Queue's code, it looks like it expects the job to be in the failed state.

Thank you @bradvogel for clarify this for Bee Queue, I was considering Bull, I added a condition to show this button only with Bull. Please when you have some time give a second look :eyes:

File: example/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const Arena = require('../');
 const Bee = require('bee-queue');
+// const Bull = require('bull');
```
Comment (roggervalf) [inline example/index.js:3]: I added Bull in example and comment those lines, so people could try bull by commenting Bee and use Bull if they want, what do you think about this?

Comment (bradvogel) [reply example/index.js:3]: Mind splitting it out to a different file? Perhaps bee.js and bull.js. That'll be more maintainable in the long run, despite the redundancy in the files today.

File: src/server/views/dashboard/templates/queueJobsByState.hbs [hunk 2]
```diff
@@ -50,12 +50,12 @@
       data-queue-state="{{ state }}" class="js-bulk-action btn btn-danger">
       Remove Jobs
     </button>
-    {{#eq state 'failed'}}
+    {{#if (or (eq state 'failed') (and (eq state 'delayed') (eq queueType 'bull')))}}
```
Comment (bradvogel) [inline src/server/views/dashboard/templates/queueJobsByState.hbs:53]: There are already variables in this file for `disablePagination` and `disableOrdering`, so what do you think about using `{{#unless disableRetry}}` here for consistency? Then you won't need this complex logic in the template file - you can push it into the server-side JS file.

**Review by bradvogel — CHANGES_REQUESTED** [review]

File: src/server/views/dashboard/templates/queueJobsByState.hbs (hunk 2 above)
Comment (roggervalf) [reply src/server/views/dashboard/templates/queueJobsByState.hbs]: sure, I updated the PR with the requested changes. Tyty

File: example/README.md [hunk 3]
```diff
@@ -13,6 +13,10 @@ This is a simple demonstration of how to run Arena and connect it to [Bee Queue]
 
 ## Running
 
-`npm start`
+`npm start:bee`
```
Comment (bradvogel) [inline example/README.md:16]: neat!

**Review by bradvogel — APPROVED** [review]

**Pull request merged**